src/Uno.UI.Wasm/WasmScripts/Uno.UI.js

```diff
@@ -823,41 +823,31 @@ var Uno;
         (function (Interop) {
             class ManagedObject {
                 static init() {
-                    if (Uno.UI.WindowManager.isHosted) {
-                        console.debug("Hosted Mode: skipping IndexDB initialization");
-                    }
-                    else {
+                    if (!ManagedObject.assembly) {
```

Comment:
Remove this file.

Reply:
Checked with @jeromelaban, no effect, we can keep it